client/src/js/services/DebtorGroupService.js

```diff
@@ -0,0 +1,28 @@
+angular.module('bhima.services')
+.service('DebtorGroupService', DebtorGroupService);
+
+DebtorGroupService.$inject = ['$http', 'util'];
+
+/**
+* Debtor Group Service
+*
+* This service implements CRUD operations for the /debtor_groups API endpoint
+*/
+function DebtorGroupService($http, util) {
+  var service = this;
+
+  /** Exposed method read */
+  service.read = read;
+
+  /**
+  * @method read
+  * @param {string} uuid The debtor group uuid
+  * @param {object} parameters The query string object
+  * @description This function is responsible for getting debtor groups
+  */
+  function read(uuid, parameters) {
+    var url = uuid ? '/debtor_groups/' + uuid : '/debtor_groups/';
```

Comment:
A more readable version of this code is this:
```js
var url = '/debtor_groups/'.concat(uuid || '');
```
Could you also move the '/debtor_groups/' url to a service variable?  Something like:
```js
var service = this;
var baseUrl = '/debtor_groups/';
```
This is so that, if we change the API, we only have to change one variable.